plus/src/test/java/org/apache/calcite/adapter/tpcds/TpcdsTest.java

```diff
@@ -164,30 +165,30 @@ private static String schema(String name, String scaleFactor) {
 
   @Test public void testTableCount() {
     final CalciteAssert.AssertThat with = with();
-//    foo(with, "CALL_CENTER", 6);
-//    foo(with, "CATALOG_PAGE", 11_718);
-//    foo(with, "CATALOG_RETURNS", 144_067);
-//    foo(with, "CATALOG_SALES", 1_441_548);
-//    foo(with, "CUSTOMER", 100_000);
-//    foo(with, "CUSTOMER_ADDRESS", 50_000);
-//    foo(with, "CUSTOMER_DEMOGRAPHICS", 1_920_800);
-//    foo(with, "DATE_DIM", 73_049);
-//    foo(with, "HOUSEHOLD_DEMOGRAPHICS", 7_200);
-//    foo(with, "INCOME_BAND", 20);
-//    foo(with, "INVENTORY", 11_745_000);
-//    foo(with, "ITEM", 18_000);
-//    foo(with, "PROMOTION", 300);
-//    foo(with, "REASON", 35);
-//    foo(with, "SHIP_MODE", 20);
-//    foo(with, "STORE", 12);
-//    foo(with, "STORE_RETURNS", 287_514);
-//    foo(with, "STORE_SALES", 2_880_404);
-//    foo(with, "TIME_DIM", 86_400);
-//    foo(with, "WAREHOUSE", 5);
-//    foo(with, "WEB_PAGE", 60);
-//    foo(with, "WEB_RETURNS", 71_763);
-//    foo(with, "WEB_SALES", 719_384);
-//    foo(with, "WEB_SITE", 30);
```

Comment:
Why to open up these tests ? which seems has no relationship with the fix title.

Reply:
Thanks for @danny0405 's replies. I just doubt why this should be commented。If it's not ok, i will keep original instead.

Reply:
I see @julianhyde did the modification in https://github.com/apache/calcite/commit/93b8349895df2183864bc4bd7ef2fafa227b22e9, maybe he can answer you!

Reply:
I would assume to avoid generating data for those tables just for the sake of count queries but I may be wrong.

Reply:
Thanks for @zabetak 's replies.  Those tables' rowCounts are added in `org.apache.calcite.adapter.tpcds.TpcdsSchema`,  it uses TABLE_ROW_COUNTS mapping to maintaince tables' rowCounts. So I think there's no cost to generate data?

Maybe it's better to open up these tests again?

Reply:
I see that `foo` method executes a query to the table passed as a parameter. If I remember well `TpcdsQueryableTable` which I think is involved on this query generates the respective data every time it is referenced.

Reply:
If those codes are useless, so just remove them any more?  I wonder what I should do, maybe just keep it original?